package.json

```diff
@@ -192,6 +192,12 @@
       }
     ],
     "menus": {
+      "commandPalette": [
+          {
+              "command": "coder.openFromSidebar",
+              "when": "false"
```

Comment:
Just real quick can you explain how this setting works? Why provide a command we don't want people to run?

Reply:
We need the command to run it in a context menu from the sidebar: https://github.com/coder/vscode-coder/blob/2d428eb88ca352519dc5be2666c6dd8eba276617/package.json#L216-L220

But, while we want it in the context menu where it has access to a selected workspace, we do not want it in the palette, where there is no selected workspace.

Reply:
Commands show up by default in the palette, so we have to explicitly prevent that with the `when`.  Feels janky to me but apparently this is the intended way lol

Reply:
Agree haha